toolkit/docs/formats/imageconfig.md

```diff
@@ -429,6 +429,15 @@ A sample KernelCommandLine enabling SELinux and booting in enforcing mode:
 },
 ```
 
+A sample KernelCommandLine enabling SELinux and overwriting the default 'selinux-policy' package name:
+
+``` json
+"KernelCommandLine": {
+    "SELinux": "enforcing",
+    "SELinuxPolicy": "my-selinux-policy"
```

Comment:
Above there are dedicated sections describing all the fields under `KernelCommandLine`. You should add one for `SELinuxPolicy` as well.

Reply:
It's just your preference, I wanted to have a sample that shows "SELinuxPolicy" is not required to be set. I stick with this.

Reply:
Please fix. All fields/options should be fully documented. If the field is optional, then state that it is optional in its documentation.

Reply:
Oops! my bad. I thought I added that section, but apparently it was missed.